indexing-service/src/test/java/io/druid/indexing/common/task/RealtimeIndexTaskTest.java

```diff
@@ -1114,6 +1120,9 @@ public long sumMetric(final Task task, final DimFilter filter, final String metr
         task.getQueryRunner(query).run(QueryPlus.wrap(query), ImmutableMap.<String, Object>of()),
         Lists.<Result<TimeseriesResultValue>>newArrayList()
     );
-    return results.isEmpty() ? 0 : results.get(0).getValue().getLongMetric(metric);
+    if (results.isEmpty()) {
+      return NullHandling.nullToZeroIfNeeded((Long) null);
```

Comment:
Suggested to extract `defaultLongValue()`

Reply:
done.